tests/spec/localhost/nvidia_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe command('nvidia-docker run --rm nvidia/cuda nvidia-smi') do
```

Comment:
yes this implementation is leaving out the fact that we might build on an instance without nvidia device.

but i really like the cuda containers `nvidia-smi` cause it handles a lot tests in one... but i'm open for suggestions